kubernetes/configs/prod/versions.sh

```diff
@@ -25,4 +25,4 @@ JOB_DETAILS_PERSISTER_IMAGE_VERSION="1.0.11"
 JOB_TENANTS_CERTIFIED_ATTRIBUTES_UPDATER_IMAGE_VERSION="1.0.11"
 JOB_METRICS_REPORT_GENERATOR_IMAGE_VERSION="1.0.11"
 JOB_PADIGITALE_REPORT_GENERATOR_IMAGE_VERSION="1.0.11"
```

Comment:
Why not moving also these versions to 1.0.12?

Reply:
makes sense